rastreio/__main__.py

```diff
@@ -27,5 +27,15 @@ def update_packages():
     update_packages.run()
 
 
+@cli.command(name="packages:delete")
+@click.option("--code", help="Package code to be deleted")
+def clean_packages(code):
+    """Delete an active package"""
```

Comment:
```suggestion
    """Delete a package"""
```

Considerando que não checamos se o pacote é ativo ou não.